data/json/npcs/valhalla_cult/missions.json

```
@@ -8,8 +8,7 @@
     "difficulty": 5,
     "value": 0,
     "start": {
-      "assign_mission_target": { "om_terrain": "field", "reveal_radius": 1, "random": true, "search_range": 20, "min_distance": 5, "z": 0 },
-      "update_mapgen": { "place_monster": [ { "monster": "mon_nakedmolerat_giant", "x": 11, "y": 11, "target": true } ] }
+      "assign_mission_target": { "om_terrain": "field", "reveal_radius": 1, "random": true, "search_range": 20, "min_distance": 5, "z": 0 }
```

```suggestion
      "assign_mission_target": { "om_terrain": "field", "reveal_radius": 1, "random": true, "search_range": 20, "min_distance": 5, "z": 0 },
      "update_mapgen": { "place_monster": [ { "monster": "mon_wolf_mutant_huge", "x": 11, "y": 11, "target": true } ] }

```
Suggest replacing monster to not break this mission chain. A quick google suggests big wolves are very important in Norse mythology, more so than giant naked rats anyway